src/Microsoft.Data.SqlClient/netfx/src/Microsoft/Data/SqlClient/SqlDataReader.cs

```diff
@@ -2894,6 +2895,19 @@ virtual public SqlXml GetSqlXml(int i)
             return sx;
         }
 
+        /// <summary>
+        /// Retrieves the column at ordinal as a SqlJson.
+        /// </summary>
+        /// <param name="i"></param>
+        /// <returns></returns>
+        virtual public SqlJson GetSqlJson(int i)
```

Comment:
Please move documentation to XML file and also include this API + doc in ref projects.